docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -19,46 +79,29 @@ Use this extension to get the full list of problems found by Codacy for a pull r
 
-To see Codacy quality and coverage data for the pull request you're working on, follow these steps:
+To see Codacy quality and coverage data for a pull request, follow these steps:
 
 1.  Open the repository directory in VS Code.
 
     !!! note
         If the repository isn't on Codacy yet, [add it to Codacy](../organizations/managing-repositories.md#adding-a-repository) first.
 
-1.  Check out the branch associated with the open pull request you're working on.
-
-1.  Open the main view by clicking the **Codacy logo** in the activity bar or the **Codacy tab** in the status bar.
+1.  Open the main view by clicking the **Codacy logo** in the activity bar.
 
     ![Codacy main view](images/codacy-vscode-extension-sign-in.png)
 
 1.  If you’re not signed in, click the **Sign in** button to authorize VS Code on Codacy.
 
-After completing these steps, the main view shows the result of the latest Codacy analysis for the pull request. The VS Code Problems tab lists the Quality issues found.
-
-![Codacy main view](images/codacy-vscode-extension-main-view.png)
-
-The main view displays the information about the code quality changes introduced by the pull request:
-
--   The **Quality status** of the pull request, either up to standards or not up to standards, based on the [Quality gates](../repositories-configure/adjusting-quality-gates.md) set for the repository.
-
--   Any **Quality issues** introduced or fixed by the pull request. These are the same issues you find in the [Quality Issues tabs](../repositories/pull-requests.md#issues-tabs) in the Codacy app and are also visible in VS Code's Problems tab.
-
-    The number next to each file name is the total number of Quality issues that the pull request adds to or removes from that file. The number farther to the right is the number of new Medium or Critical Quality issues. If there are any such issues, the file name is also highlighted in yellow (Medium) or red (Critical).
-
-    ![Important issues](./images/codacy-vscode-extension-important-issues.png)
-
--   The **diff coverage and coverage variation** introduced by the pull request. These are the same values you find in the [Pull request coverage overview](../repositories-coverage/pull-requests.md#coverage-overview) panel in the Codacy app.
-
-    The percentage next to each file name is the coverage variation for that file.
+    !!! tip
+        You can alternatively sign in, and access more Codacy commands, such as **Sign out**, by opening the VS Code command palette (`Ctrl+Shift+P` on Windows/Linux or `Cmd+Shift+P` on macOS) and typing "Codacy".
 
--   Sequences of **duplicate code** (clones). These are the same ones you find in the [Quality Duplication tabs](../repositories/pull-requests.md#duplication-tabs) in the Codacy app.
+1.  Checkout the pull request of interest, either manually or from the **Open Pull Requests** tab, by clicking the arrow button or using the contextual right-click menu.
```

Comment:
Tweaks:
- Breaking the sentence, for clarity.
- Changing the check-out object to "branch". Please evaluate. 🙏 

```suggestion
1.  Check out the branch associated with the pull request of interest. You can do it either manually or from the **Open Pull Requests** tab, by clicking the arrow button or using the contextual right-click menu.
```

Reply:
OK to break up the sentence, with two tweaks:

* I'd still refer to a "pull request", see my previous comment
* It seems that "checkout" is also commonly used. It's a subtle difference, and both are considered valid in different contexts. We can use "check out" here since we're referring to a PR. But if we decide to refer to a branch, "checkout" would make more sense IMO, as it's closer to the Git command.

```suggestion
1.  Check out the pull request of interest. You can do it either manually or from the **Open Pull Requests** tab, by clicking the arrow button or using the contextual right-click menu.
```